cypress/support/pages/signUp.pageObject.js

```diff
@@ -0,0 +1,55 @@
+import PageObject from '../PageObject';
+
+class SignUpPageObject extends PageObject {
+  url = '/#/register';
+
+  get usernameField() {
+    return cy.getByDataCy('username-sign-up');
+  }
+
+  get emailField() {
+    return cy.getByDataCy('email-sign-up');
+  }
+
+  get passwordField() {
+    return cy.getByDataCy('password-sign-up');
+  }
+
+  get signUpBtn() {
+    return cy.getByDataCy('sign-up-btn');
+  }
+
+  get modalTitle() {
+    return cy.get('.swal-title');
+  }
+
+  typeUsername(username) {
+    this.usernameField
+      .type(username);
+  }
+
+  typeEmail(email) {
+    this.emailField
+      .type(email);
+  }
+
+  typePassword(password) {
+    this.passwordField
+      .type(password);
+  }
+
+  clickSignUpBtn() {
+    this.signUpBtn
+      .click();
+  }
+
+  checkSuccess(text) {
+    this.modalTitle.should('contain.text', text);
```

Comment:
Verify that the `checkSuccess` method correctly checks for the success message in the modal title. Ensure that the modal title element is correctly selected and contains the expected text.